### PR TITLE
Turn off spatial

### PR DESCRIPTION
**Pull request by popkinj**
Turned off all spatial logic in the table inserts. It's currently failing and slowing up the team.
Will turn back on in a separate branch.

### CODE REVIEW

File: api-mobile/.gitignore [hunk 1]
```diff
@@ -10,3 +10,5 @@ dist
 coverage
 
 .env
+env.local
```
Comment (NickPhura) [inline api-mobile/.gitignore]: I don't think we want to delete/ignore this file?

File: api-mobile/src/swagger/swagger.yaml [hunk 2]
```diff
@@ -1,911 +0,0 @@
-swagger: "2.0" # npm library `swagger-tools` does not yet support 3.0
```
Comment (NickPhura) [inline api-mobile/src/swagger/swagger.yaml]: Definitely dont want to delete this file

File: api-mobile/.gitignore (hunk 1 above)
Comment (popkinj) [reply api-mobile/.gitignore]: Yeah.. was thinking of this.
The problem is that it now contains credentials.

Comment (popkinj) [reply api-mobile/.gitignore]: Real ones.

Comment (NickPhura) [reply api-mobile/.gitignore]: does it? I think thats just the default for a default postgres setup?

Comment (NickPhura) [reply api-mobile/.gitignore]: The credentials we use in openshift are different

Comment (NickPhura) [reply api-mobile/.gitignore]: All of the `**/env_config/*` files are only used in local setups, they aren't used in openshift for any of the real deployments.  Or at the very least, the DB username/password ones get overwritten with openshift secrets

File: api-mobile/src/swagger/swagger.yaml (hunk 2 above)
Comment (popkinj) [reply api-mobile/src/swagger/swagger.yaml]: Oo. yeah. Don't know why that happened.

Comment (popkinj) [reply api-mobile/src/swagger/swagger.yaml]: I'll fix that first.

File: api-mobile/.gitignore (hunk 1 above)
Comment (popkinj) [reply api-mobile/.gitignore]: But doesn't the make process always overwrite .env with env.local?

Comment (popkinj) [reply api-mobile/.gitignore]: The instructions in the readme say to copy the credentials into it.

File: api-mobile/src/swagger/swagger.yaml (hunk 2 above)
Comment (popkinj) [reply api-mobile/src/swagger/swagger.yaml]: Swagger is including in the top level .gitignore. That's weird.

Comment (popkinj) [reply api-mobile/src/swagger/swagger.yaml]: k... removed swagger from the `.gitignore`. It gets confusing having all these different copies.

File: api-mobile/.gitignore (hunk 1 above)
Comment (popkinj) [reply api-mobile/.gitignore]: K... I'll just reverse this as well. I just put the credentials into my shell config.

Comment (NickPhura) [reply api-mobile/.gitignore]: Its a bit confusing, but long story kind of short:

The main purpose of the makefile is just to facilitate the local docker setup.  
When you run `make local` it runs all of the commands needed to build/run docker locally.

The `make setup-local` and `env.local` were entirely just there as a bit of convenience when running the app locally. The only real difference between `env.local` and `env.docker` is the `DB_HOST` variable.

But, I've just realized that the api-mobile code never actually reads from the `.env` file.  So currently `make setup-local` doesn't really help with anything. But that being said, I am adding a tiny bit of code to api-mobile that will actually read the `.env`.  And then you will be able to run `make setup-local` and then edit the `.env` as needed to match your local postgres username/password, and then it should be happy.

File: api-mobile/src/swagger/swagger.yaml (hunk 2 above)
Comment (NickPhura) [reply api-mobile/src/swagger/swagger.yaml]: Oh thats weird

File: api-mobile/.gitignore (hunk 1 above)
Comment (popkinj) [reply api-mobile/.gitignore]: Sounds good @NickPhura

**Review by NickPhura — APPROVED** [review]

**Pull request merged**